### PR TITLE
feat: add `--use_python_runtime` and `--enable_cuda_graph` args to the perf run script

### PR DESCRIPTION
**Pull request by zewenli98**
# Description

Add `--use_python_runtime` and `--enable_cuda_graph` args to the perf run script.

## Type of change

- New feature (non-breaking change which adds functionality)

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

File: tools/perf/perf_run.py [hunk 1]
```diff
@@ -653,6 +667,8 @@ def run(
             "No Pytorch model (nn.Module) is provided for torchdynamo compilation. Please provide a pytorch model using --model_torch argument"
         )
 
+    torchtrt.runtime.set_cudagraphs_mode(params.get("enable_cuda_graph", False))
```
Comment (peri044) [inline tools/perf/perf_run.py:670]: This will only enable cuda graphs for TRT modules. If you want the cuda graphs for whole model (in case of fallback), you need to use with context manager I think.

**Review by peri044 — APPROVED** [review]

File: tools/perf/perf_run.py [hunk 2]
```diff
@@ -328,16 +382,29 @@ def run_hf_inductor(model, input_tensors, params, precision, batch_size):
...
+        record_llm_perf(
+            model,
+            "Inductor",
+            input_tensors,
+            precision,
+            osl,
+            batch_size,
+            iters,
+            compile_time_s,
+        )
```
Comment (peri044) [inline tools/perf/perf_run.py:407]: This option doesn't apply to inductor, it is only for Torch-TRT (dynamo/torch.compile backend)

File: tools/perf/perf_run.py [hunk 3]
```diff
@@ -363,9 +430,27 @@ def run_inductor(model, input_tensors, params, precision, batch_size):
...
+    else:
+        record_perf(
+            model,
+            "inductor",
+            input_tensors,
+            precision,
+            iters,
+            batch_size,
+            compile_time_s,
+        )
```
Comment (peri044) [inline tools/perf/perf_run.py:453]: This option doesn't apply to inductor, it is only for Torch-TRT (dynamo/torch.compile backend)

File: tools/perf/perf_run.py [hunk 4]
```diff
@@ -187,15 +188,27 @@ def run_ts_trt(model, input_tensors, params, precision, batch_size):
...
+    else:
+        record_perf(
+            model,
+            "Torchscript",
+            input_tensors,
+            precision,
+            iters,
+            batch_size,
+            compile_time_s,
+        )
```
Comment (peri044) [inline tools/perf/perf_run.py:211]: This option doesn't apply to torchscript, it is only for Torch-TRT (dynamo/torch.compile backend)

**Review by peri044 — APPROVED** [review]
LGTM
